src/containers/FrontpageEditPage/FrontpageNodeList.tsx

```diff
@@ -70,9 +72,10 @@ const FrontpageNodeList: ComponentType<Props> = ({ name, replace, remove, level,
             replace={replace}
             index={index}
             level={level}
+            shouldIndent={menuField.value.length < FRONTPAGE_DEPTH_LIMIT}
```

Comment:
Burde denne logikken utføres inne i FrontpageNode?

Reply:
Altså at jeg heller burde sende ned menuField for å gjøre logikken i FrontpageNode? Isåfall tenker jeg nei 🤔

Reply:
Kan ikke dette utledes fra `level`?

Reply:
Ser ikke helt hvordan? 🤔 men åpen for innspill på det om du ser noe lurt altså!